bukuserver/translations/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,552 @@
+# German translations for bukuserver.
+# Copyright (C) 2024 buku
+# This file is distributed under the same license as the buku project.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version:  4.9\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2024-09-12 00:00+0000\n"
+"PO-Revision-Date: 2024-09-08 19:42+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: de\n"
```

Comment:
`de`/`fr` translation files were added but only a few strings were translated in them